_inc/client/traffic/verification-services.jsx

```diff
@@ -123,20 +125,41 @@ export const VerificationServices = moduleSettingsForm(
 							{ __( 'Generate XML sitemaps' ) }
 						</ModuleToggle>
 						{
-							this.props.getOptionValue( 'sitemaps' ) && (
-								<FormFieldset>
-									<p>
-										<ExternalLink icon={ true } target="_blank" href={ sitemap_url }>{ sitemap_url }</ExternalLink>
-										<br />
-										<ExternalLink icon={ true } target="_blank" href={ news_sitemap_url }>{ news_sitemap_url }</ExternalLink>
+							this.props.isSiteVisibleToSearchEngines
+								? this.props.getOptionValue( 'sitemaps' ) && (
+									<FormFieldset>
+										<p>
+											<ExternalLink icon={ true } target="_blank" href={ sitemap_url }>{ sitemap_url }</ExternalLink>
+											<br />
+											<ExternalLink icon={ true } target="_blank" href={ news_sitemap_url }>{ news_sitemap_url }</ExternalLink>
+										</p>
+										<p className="jp-form-setting-explanation">{ __( 'Your sitemap is automatically sent to all major search engines for indexing.' ) }</p>
+									</FormFieldset>
+								)
+								: (
+									<p className="jp-form-setting-explanation">
+										{
+											__( 'Your site must be accessible by search engines for this feature to work properly. You can change this in {{a}}Reading Settings{{/a}}.', {
```

Comment:
This may not be the only reason a site is not public.  Beau just ran into that the other day when his robots.txt was disallowed.  I think we might just want to hide this card completely, or simply remove the suggestions on how to fix.